### PR TITLE
[AR Number] Improvements to number model and 5 issues fixed

### PR DESCRIPTION
**Pull request by MuhammedSalamneu**
Fixed 5 issues in numbermodel existing
اشتريتُ طاولتين اثنتين
مائة و ستة عشر
رأيتُ اثنتين و خمسين بنتا
مائة واثنتين وثلاثين على خمس

Regex: TenToNineteenIntegerRegex and ZeroToNineIntegerRegex.

- [x] No duplicate codes? 

- [x] No build error? 

- [x] Ran all test cases? 

- [x] All cases getting passed? 

- [x] Followed reusability? 

- [x] Followed the naming conventions? 

- [x] New change does not negatively impact on performance? 

- [x] Is the code readable? 

- [x] Following the best practices?

### CODE REVIEW

File: Specs/Number/Arabic/NumberRangeModel.json [hunk 1]
```diff
@@ -716,8 +716,8 @@
   },
   {
     "Input": "يساوي ٥٠٠٠ او اكثر من",
-    "IgnoreResolution": true,
     "NotSupportedByDesign": "java, javascript, python",
+    "NotSupported": "dotnet",
```
Comment (tellarin) [inline Specs/Number/Arabic/NumberRangeModel.json:720]: Why is this now not supported if it worked before?

File: Specs/Number/Arabic/NumberRangeModel.json [hunk 2]
```diff
@@ -1291,6 +1281,366 @@
...
+        },
+        "Start": 6,
+        "End": 27
+      }
+    ]
+  },
+  {
+    "Input": "درجته تساوي ٣٠ أو ما تقل عنه",
+    "NotSupportedByDesign": "java, javascript, python",
+    "NotSupported": "dotnet",
```
Comment (tellarin) [inline Specs/Number/Arabic/NumberRangeModel.json:1414]: Why are these new cases not supported?

File: Patterns/Arabic/Arabic-Numbers.yaml [hunk 3]
```diff
@@ -639,4 +645,7 @@ RelativeReferenceRelativeToMap: !dictionary
     الذي قبلا الأخير: end
     الثانية الى الاخير: end
     السابق: current    
-...
\ No newline at end of file
+...
+
+
```
Comment (tellarin) [inline Patterns/Arabic/Arabic-Numbers.yaml:650]: Minor: Please leave only one blank line at the end of the file

File: Specs/Number/Arabic/NumberRangeModel.json (hunk 2 above)
Comment (neudurgeshp) [reply Specs/Number/Arabic/NumberRangeModel.json:1414]: @tellarin it was for our Neudesic branch. He (@MuhammedSalamneu) raised to wrong target by mistake.
@MuhammedSalamneu please close this PR.

File: Patterns/Arabic/Arabic-Numbers.yaml (hunk 3 above)
Comment (neudurgeshp) [reply Patterns/Arabic/Arabic-Numbers.yaml:650]: @tellarin it was for our Neudesic branch. He (@MuhammedSalamneu) raised to wrong target by mistake.
@MuhammedSalamneu please close this PR.

**Pull request closed without merging**